client/src/components/exercise/lab9/Lab9Context.js

```diff
@@ -0,0 +1,8 @@
+import { createContext } from "react";
+
+const GameStateContext = createContext({
```

Comment:
If we want to leave lab 9 out of this PR we probably don't need this

Reply:
i figured that having some starter code for everyone in terms of the new code block and lab 9 context implementation would make things easy to build off of. Up to you guys tho @all-rit/developers